pkg/redact/redact.go

```diff
@@ -32,5 +32,9 @@ func Key(key []byte) string {
 	if NeedRedact() {
 		return "?"
 	}
+	if len(key) == 0 {
+		// TiKV treats zero length keys as infinate(i.e. unbounded).
+		return "inf"
+	}
```

Comment:
looks unnecessary

```suggestion
```

Reply:
Yep, identify it was `+inf` or `-inf` is impossible in this context... Remote it is acceptable.